### PR TITLE
feat(dropdown) Add invalid and invalidText inputs

### LINKED ISSUE #785: Dropdown - Add invalid and invalidText inputs as in ibm-label inputs
## Detailed description
This is a feature request, to have available the possibility to set in an easy way the error messages and invalid state to dropdowns just as the inputs.

### PR DESCRIPTION
**Pull request by Donisius**
Closes #785

### CODE REVIEW

File: src/dropdown/dropdown.stories.ts [hunk 1]
```diff
@@ -126,14 +134,19 @@ storiesOf("Dropdown", module)
 		</div>
 		<ibm-placeholder></ibm-placeholder>
 	`,
-		props: getProps()
+		props: getProps({
+			invalid: boolean("Invalid", false),
+			invalidText: "This is not a validation text"
```
Comment (cal-smith) [inline src/dropdown/dropdown.stories.ts]: You don't need to add that here if it's already in `getProps` 👍

File: src/dropdown/dropdown.component.ts [hunk 2]
```diff
@@ -93,6 +94,10 @@ import { scrollableParentsObservable, isVisibleInContainer } from "./../utils/sc
 				[ngTemplateOutletContext]="getRenderTemplateContext()"
 				[ngTemplateOutlet]="displayValue">
 			</ng-template>
+			<ibm-icon-warning-filled16
```
Comment (cal-smith) [inline src/dropdown/dropdown.component.ts]: Can we use the directive form of the icon instead? I'm noticing some weird alignment issues ... so it'd be 

```html
<svg ibmIconWarningFilled16
    *ngIf="invalid"
    class="bx--dropdown__invalid-icon">
</svg>

**Comment by Donisius** [comment]
> Could you ammend your commit to be `feat(dropdown) Add invalid and invalidText inputs` ... I was super confused until I looked at the code 😅

**Comment by cal-smith** [comment]
@Donisius any reason you closed this?

**Comment by Donisius** [comment]
@cal-smith My mistake, didnt realize I did.

**Pull request merged**